### PR TITLE
Remove WindowOrWorkerGlobalScope [es]

### PR DESCRIPTION
**Pull request by Graywolf9**
### Description

Remove WindowOrWorkerGlobalScope page and mentions for `es`

Steps:
- [x] Remove the page for `Web/API/WindowOrWorkerGlobalScope`
- [x] Search for pages with a title that starts with `WindowOrWorkerGlobalScope.` -- for each page:
  - [ ] Move the page a folder up using `yarn content move Web/API/WindowOrWorkerGlobalScope/<prop> Web/API/<prop> <locale>` in `mdn-content` (already moved)
  - [x] Update the title to remove mention (AKA, `WindowOrWorkerGlobalScope.caches` becomes `caches`)
  - [ ] Update the `translation_of` key to remove mention (AKA, `Web/API/WindowOrWorkerGlobalScope/caches` becomes `Web/API/caches`) (there wasn't)
  - [ ] Update `browser-compat` key to remove mention (AKA, `api.WindowOrWorkerGlobalScope.caches` becomes `api.caches`) (there wasn't)
    - If there is no `browser-compat` key, but there is a `{{Compat}}` macro, update its value instead
    - If neither a `browser-compat` key or `{{Compat}}` macro call exist, ignore this step
- [x] Remove all `{{domxref}}` references to `WindowOrWorkerGlobalScope` (the regex `domxref[ ]*\([ ]*["'][ ]*WindowOrWorkerGlobalScope` may help find these instances)
- [x] Remove all the links mentioning

### Motivation

Fix an issue

### Related issues and pull requests

Relates to #5617

### CODE REVIEW

File: files/es/web/api/settimeout/index.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-title: WindowOrWorkerGlobalScope.setTimeout
+title: setTimeout
 slug: Web/API/setTimeout
 original_slug: Web/API/WindowOrWorkerGlobalScope/setTimeout
 ---
 
 {{APIRef("HTML DOM")}}
 
-El método **`setTimeout()`** del mixin {{domxref("WindowOrWorkerGlobalScope")}} establece un temporizador que ejecuta una función o una porción de código después de que transcurre un tiempo establecido.
+El método **`setTimeout()`**  establece un temporizador que ejecuta una función o una porción de código después de que transcurre un tiempo establecido.
```
Comment (Graywolf9) [inline files/es/web/api/settimeout/index.md:9]:
```suggestion
El método **`setTimeout()`** establece un temporizador que ejecuta una función o una porción de código después de que transcurre un tiempo establecido.
```

File: files/es/web/api/indexeddb/index.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
 ---
-title: WindowOrWorkerGlobalScope.indexedDB
+title: indexedDB
 slug: Web/API/indexedDB
 original_slug: Web/API/WindowOrWorkerGlobalScope/indexedDB
```
Comment (JuanVqz) [inline files/es/web/api/indexeddb/index.md:4]: Hey, @Graywolf9 thanks for contributing, should the original slug still calling the window or worker global scope?

Comment (Graywolf9) [reply files/es/web/api/indexeddb/index.md:4]: Thanks for the review! I had the same doubt but there is other issue about the front matter keys that says that it is a front matter added from Yari so it updates as necessary for it

Comment (JuanVqz) [reply files/es/web/api/indexeddb/index.md:4]: but even that didn't you think we should remove it to avoid confusion in possible next reviews?

**Review by JuanVqz — APPROVED** [review]
Anyway, approved!

File: files/es/web/api/indexeddb/index.md (hunk 2 above)
Comment (Graywolf9) [reply files/es/web/api/indexeddb/index.md:4]: https://github.com/mdn/translated-content/issues/5617#issuecomment-1153824408

**Pull request merged**